lib/ohai/dsl/plugin/versionvii.rb

```diff
@@ -195,8 +195,6 @@ def fetch_plugin_config
           # key? to avoid falsely instantiating a configuration hash.
           if Ohai.config[:plugin].key?(snake_case_name)
             Ohai.config[:plugin][snake_case_name]
-          else
-            nil
```

Comment:
If the caller is doing`.nil?` or similar on the return value this change could break it. Something like this would preserve the return value and probably fix the warning:

```
return unless Ohai.config[:plugin].key?(snake_case_name)
Ohai.config[:plugin][snake_case_name]
```

Reply:
After a slack convo it looks like we're good here since it returned nil before.